metrics/grafana/pd.json

```diff
@@ -9313,10 +9313,11 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(pd_client_request_handle_requests_duration_seconds_count{tidb_cluster=\"$tidb_cluster\"}[1m])) by (type)",
+              "expr": "sum(rate(pd_server_handle_tso_duration_seconds_count{tidb_cluster=\"$tidb_cluster\"}[1m])) by (type)",
               "format": "time_series",
+              "interval": "",
               "intervalFactor": 2,
-              "legendFormat": "{{type}}",
+              "legendFormat": "tso rps",
```

Comment:
The concept of RPS is a bit vague here, I think we should call it the number of TSO gRPC requests per second in case someone mistakes it for the actual number of TSO requests.